src/locales/en/home.json

```diff
@@ -7,6 +7,7 @@
   "models": "Our Models",
   "allModels": "All models",
   "readMore": "Read more",
+  "aboutUs": "About us",
```

Comment:
check for this field in already created files, don't create duplications. 
same for other translations.

Reply:
ok, i got it